### PR TITLE
Deploy to pypi on merge into master

### LINKED ISSUE #27: Release new version with Redis
I see there are commits for adding support for Redis (and ElasticSearch as well).
https://github.com/testcontainers/testcontainers-python/commit/f6bec8084c8cb0758de0ec4389cca00b30a71b51

When are you going to release a new version to PyPi?

### PR DESCRIPTION
**Pull request by tillahoffmann**
This PR 

* adds functionality to deploy to pypi when the master branch is built.
* bumps the minor version number.
* fixes some links in the README.

@SergeyPirogov, as I'm not a maintainer on pypi, I can't add my own encrypted password to deploy. Two options: (a) you can add me as a maintainer and I can deploy using my account, or (b) you can follow the instructions [here](https://docs.travis-ci.com/user/deployment/pypi/) to encrypt your password and add the password in the `.travis.yml` file.

This should also resolve #27.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,3 +29,11 @@ notifications:
   email:
     recipients:
       - sergio_89@ukr.net
+
+deploy:
+  provider: pypi
+  user: SergeyPirogov
+  password:
+    secure: encrypted-password
```
Comment (SergeyPirogov) [inline .travis.yml]: ok, where to get this password?

Comment (tillahoffmann) [reply .travis.yml]: This is the password you use to log into http://pypi.org.

Comment (SergeyPirogov) [reply .travis.yml]: I think we have to deploy on tag, it will give more control! I do same for other project

**Comment by SergeyPirogov** [comment]
@tillahoffmann, please give me your pypi id, so I will be able to add you as a maintainer

**Comment by tillahoffmann** [comment]
My pypi id is `tillahoffmann` (profile [here](https://pypi.org/user/tillahoffmann/)).

**Comment by tillahoffmann** [comment]
Hi @SergeyPirogov, just wanted to follow up on being added as a maintainer.

**Comment by SergeyPirogov** [comment]
@tillahoffmann added you as collaborator

**Comment by tillahoffmann** [comment]
Great, thanks for setting that up. I think this is ready to be merged.

**Pull request merged**